### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by aekong**
- added a bunch of cards
  - resource
  - topic
  - service
  - news
  - event

- removed margins on card descriptions
- added gulp-rename package to flatten image folder

future iterations on cards my change, so treating them as separate components even though there are similarities in markup and style properties

### CODE REVIEW

File: src/components/02-cards/event/_event.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.sfgov-event-card {
+  @include rubik;
+  background: $green-3;
+  border: 0;
+  border-radius: 8px;
```
Comment (jrubenoff) [inline src/components/02-cards/event/_event.scss:5]: I have a `$radius` variable for this in `00-design-tokens/_variables`

File: src/components/02-cards/event/event.hbs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<a class="sfgov-event-card" href="{{ url }}">
+  <div class="event-image" style="background-image:url(/images/event-image.png)"></div>
```
Comment (jrubenoff) [inline src/components/02-cards/event/event.hbs:2]: Any reason these aren't `img` tags?

**Review by jrubenoff — COMMENTED** [review]
Left a few comments.

If you're sure the card styles shouldn't be consolidated into mixins, this looks OK to me otherwise!

File: src/components/02-cards/event/event.hbs (hunk 2 above)
Comment (aekong) [reply src/components/02-cards/event/event.hbs:2]: no good reason, i guess.  will fix

File: src/components/02-cards/event/_event.scss (hunk 1 above)
Comment (aekong) [reply src/components/02-cards/event/_event.scss:5]: missed that, thanks

**Review by jrubenoff — APPROVED** [review]
LGTM

**Pull request merged**